backend/api_keys.ts

```diff
@@ -0,0 +1,186 @@
+import { CreateApiKeyRequest } from "@runtimed/extensions/providers/api_key";
+import * as jose from "jose";
+import {
+  RuntError,
+  ErrorType,
+  Passport,
+  AuthType,
+  AuthenticatedProviderContext,
+} from "@runtimed/extensions";
```

Comment:
I see you had to use `@runtimed` for npm whereas we have `@runt` on jsr.io. 🙃

Reply:
yeah, bridge to cross at a later time if need be